### PR TITLE
 KHP3-6497: Rename Key population to key and vulnerable population 

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

KeyP

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-care-panel-app/src/care-programs/care-programs.component.tsx [hunk 1]
```diff
@@ -78,9 +78,12 @@ const CarePrograms: React.FC<CareProgramsProps> = ({ patientUuid }) => {
   const rows = useMemo(
     () =>
       carePrograms.map((careProgram) => {
+        const displayName =
+          careProgram.display === 'Key Population' ? 'Key and Vulnerable Population' : careProgram.display;
+
```
Comment (makombe) [inline packages/esm-care-panel-app/src/care-programs/care-programs.component.tsx:83]: @its-kios09 I doubt if this check will work. We had renamed the program name from backend to be `KVP`

**Review by makombe — CHANGES_REQUESTED** [review]

File: packages/esm-care-panel-app/src/care-programs/care-programs.component.tsx (hunk 1 above)
Comment (makombe) [reply packages/esm-care-panel-app/src/care-programs/care-programs.component.tsx:83]: Build the latest module for kenyakeypop and test again

Comment (donaldkibet) [reply packages/esm-care-panel-app/src/care-programs/care-programs.component.tsx:83]: @its-kios09 any update on this PR

**Comment by donaldkibet** [comment]
@makombe mind giving this another quick review

File: packages/esm-care-panel-app/src/care-programs/care-programs.component.tsx (hunk 1 above)
Comment (its-kios09) [reply packages/esm-care-panel-app/src/care-programs/care-programs.component.tsx:83]: @donaldkibet  Nop i believed it was sorted on the backend

**Comment by its-kios09** [comment]
> @makombe mind giving this another quick review

@donaldkibet  forgot to give an update it was sorted by makombe can I close the PR?

**Comment by makombe** [comment]
@its-kios09 I think we should close this as it had been sorted at the metadata level. Thanks

**Comment by makombe** [comment]
For reference purposes https://github.com/palladiumkenya/openmrs-module-kenyakeypop/pull/206

**Pull request closed without merging**